### PR TITLE
feat(website/header): header redesign

### PR DESCRIPTION
**Pull request by tiloyi**
## I have read [the contributing guidelines](https://mozaic.adeo.cloud/Contributing/)

- [x] Yes
- [ ] No

## Does this PR introduce a breaking change?

read [What is a breaking change ?](https://mozaic.adeo.cloud/Contributing/Prerequisite/GitConventions/#breaking-changes-)

- [ ] Yes
- [x] No

## Describe the changes

Header redesign

Github issue number or Jira issue URL: https://design-system-adeo.atlassian.net/browse/MZC-248

## Other informations
Screen Shot 2019-12-14 at 10 30 59 AM

### CODE REVIEW

File: packages/gatsby-theme-styleguide/src/gatsby-components/PageTabs/PageTabs.js [hunk 1]
```diff
@@ -8,30 +8,49 @@ const Tabs = styled.div`
   width: 100%;
   z-index: 1;
   top: 0;
-  background: #fff;
-  border-bottom: solid 1px #f5f5f5;
+  border-bottom: solid 1px #000;
+  background: #e6e6e6;
+
+  .page-tabs {
```
Comment (gael-boyenval) [inline packages/gatsby-theme-styleguide/src/gatsby-components/PageTabs/PageTabs.js]: What is that ?

File: packages/gatsby-theme-styleguide/src/gatsby-components/PageTabs/PageTabs.js [hunk 2]
```diff
@@ -65,11 +84,15 @@ class PageTabs extends PureComponent {
     const cleanTabs = this.orderPageTab(samePageTabs)
 
     return (
-      <Tabs id="page_tabs_menu">
-        <TabsWrapper>
+      <Tabs id="page_tabs_menu" className="page-tabs">
```
Comment (gael-boyenval) [inline packages/gatsby-theme-styleguide/src/gatsby-components/PageTabs/PageTabs.js]: ?

File: packages/styles/components/_c.toggle.scss [hunk 3]
```diff
@@ -18,7 +18,7 @@
     user-select: none;
```
Comment (gael-boyenval) [inline packages/styles/components/_c.toggle.scss]: not the right PR... Rebase ?

File: packages/gatsby-theme-styleguide/src/templates/pattern-page.js [hunk 4]
```diff
@@ -14,7 +14,25 @@ const FullWidthContainer = styled.div`
   ${({ separator }) =>
     separator &&
     css`
-      border-bottom: solid 1px #f5f5f5;
+      .header {
```
Comment (gael-boyenval) [inline packages/gatsby-theme-styleguide/src/templates/pattern-page.js]: seriously ? WTF ? 
https://www.styled-components.com

File: packages/gatsby-theme-styleguide/src/gatsby-components/PageTabs/PageTabs.js [hunk 5]
```diff
@@ -65,11 +84,15 @@ class PageTabs extends PureComponent {
...
+        <TabsWrapper className="page-tabs__content">
```
Comment (gael-boyenval) [inline packages/gatsby-theme-styleguide/src/gatsby-components/PageTabs/PageTabs.js]: RTFM : https://www.styled-components.com

File: packages/gatsby-theme-styleguide/src/gatsby-components/PageTabs/PageTabs.js (hunk 1 above)
Comment (tiloyi) [reply packages/gatsby-theme-styleguide/src/gatsby-components/PageTabs/PageTabs.js]: fixed

File: packages/gatsby-theme-styleguide/src/gatsby-components/PageTabs/PageTabs.js (hunk 2 above)
Comment (tiloyi) [reply packages/gatsby-theme-styleguide/src/gatsby-components/PageTabs/PageTabs.js]: fixed

File: packages/styles/components/_c.toggle.scss (hunk 3 above)
Comment (tiloyi) [reply packages/styles/components/_c.toggle.scss]: fixed

File: packages/gatsby-theme-styleguide/src/templates/pattern-page.js (hunk 4 above)
Comment (tiloyi) [reply packages/gatsby-theme-styleguide/src/templates/pattern-page.js]: fixed

File: packages/gatsby-theme-styleguide/src/gatsby-components/PageTabs/PageTabs.js (hunk 5 above)
Comment (tiloyi) [reply packages/gatsby-theme-styleguide/src/gatsby-components/PageTabs/PageTabs.js]: fixed

**Pull request merged**